src/main/java/com/google/sps/model/TasksClientImpl.java

```diff
@@ -46,7 +46,8 @@ private TasksClientImpl(Credential credential) {
   @Override
   public List<Task> listTasks(TaskList taskList) throws IOException {
     // returns null if no tasks exist. Convert to empty list for ease.
-    List<Task> tasks = tasksService.tasks().list(taskList.getId()).execute().getItems();
+    List<Task> tasks =
+        tasksService.tasks().list(taskList.getId()).setShowHidden(true).execute().getItems();
```

Comment:
Please update the documentation of the `TasksClient` interface to indicate that hidden/completed tasks are being returned as well. That's not immediately obvious. Also add a comment here explaining what `setShowHidden` does.

Reply:
Fair enough, thanks!

Reply:
`RESOLVED`